app/models/article.rb

```diff
@@ -28,10 +28,11 @@ class Article < ApplicationRecord
     validates :article_tags, presence: true
   end
 
-  scope :released, -> { where.not(shipped_at: nil) }
+  scope :shipped, -> { where.not(shipped_at: nil) }
+  scope :wip, -> { where(shipped_at: nil) }
 
-  def shipped!
-    self.shipped_at = Time.now
+  def ship
```

Comment:
__[method_definition_validator]__
* shipped! is undefined. Did you mean? ship at [app/controllers/monthly_reports_controller.rb#L79](/hr-dash/hr-dash/blob/df42a105ca99a04d9e381fa7c46881ce772a4311/app/controllers/monthly_reports_controller.rb#L79)
* shipped! is undefined. Did you mean? ship at [spec/models/monthly_report_spec.rb#L164](/hr-dash/hr-dash/blob/df42a105ca99a04d9e381fa7c46881ce772a4311/spec/models/monthly_report_spec.rb#L164)
* shipped! is undefined. Did you mean? ship at [spec/models/monthly_report_spec.rb#L175](/hr-dash/hr-dash/blob/df42a105ca99a04d9e381fa7c46881ce772a4311/spec/models/monthly_report_spec.rb#L175)

<!-- Comment by [SideCI](https://sideci.com) -->